src/components/JoinPage/styles.css

```diff
@@ -0,0 +1,58 @@
+* {
+    box-sizing: border-box;
+  }
+
+body {
+    font-family: 'PT Sans', sans-serif;
+    background-image: url("../../assets/myWord_Background.jpg");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: center 1px; 
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+.card {
+  margin: auto;
+  margin-top: 23px;
+  text-align: center  !important;
+  border-radius: 25px !important;
+}
+
+.cardHeader {
+  background-color:#74aec1 !important;
+  margin-top: 90px;
+  text-align: center;
+  margin: 0 auto;
+  border-radius: 25px !important;
+}
+
+.card-header {
+  background-color: #c8a885 !important;
+  color: white;
+  font-weight: 600;
+  font-size: x-large;
+  border-top-left-radius: 25px !important;
+  border-top-right-radius: 25px !important;
+  border-color:#c8a885 !important;
```

Comment:
if it's possible to avoid using these `!important`s by using more specific selectors, that would be a better practice! `:)`

Reply:
Great! I removed all of the !importants, except for the border radius styles. Let me know if that is ok with you!